web/parser.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func Parser() {
```

Comment:
```suggestion
func Parser() error {
```

Instead of fmt.Println errors return them instead and you can track them where this will be used

Reply:
done